doc/source/python/python_wrapping_docker.md

```diff
@@ -1,19 +1,19 @@
 # Packaging a Python model for Seldon Core using Docker
```

Comment:
Most of the auto-formatting is due to using Prettier.  Can this just be an enforced pre-commit hook?